sacc/tracers/clusters.py

```diff
@@ -1,12 +1,14 @@
-from .base import BaseTracer
+from .base import BaseTracer, MULTIPLE_OBJECTS_PER_TABLE
 from astropy.table import Table
 
-class BinZTracer(BaseTracer, tracer_type="bin_z"):  # type: ignore
```

Comment:
I have a similar question as above, why does a class `A` need an attribute that specifies `type="a"`